bentoml/_internal/yatai_client/__init__.py

```diff
@@ -25,6 +25,7 @@
 from rich.progress import TimeElapsedColumn
 from rich.progress import TimeRemainingColumn
 from rich.progress import TransferSpeedColumn
+from typing_extensions import Literal
```

Comment:
We remove typing_extensions from out dependency, lets make this inside TYPE_CHECKING

with `from __future__ import annotations`